container_pipeline/pipeline.py

```diff
@@ -75,6 +50,11 @@ def create_project(appid, jobid, repo_url, repo_branch, repo_build_path,
      target_file, notify_email, desired_tag, depends_on,
      test_tag) = sys.argv[1:]
     try:
-        create_project(*sys.argv[1:])
+        trigger_dockerfile_linter(appid, jobid, repo_url, repo_branch,
+                                  repo_build_path, target_file, notify_email,
+                                  desired_tag, depends_on, test_tag)
+        create_project(appid, jobid, repo_url, repo_branch, repo_build_path,
+                       target_file, notify_email, desired_tag, depends_on,
+                       test_tag)
     except OpenshiftError:
```

Comment:
@dharmit yes, that is exactly what I meant. we should create the project in OpenShift only if linter succeeds or at least triggered after dockerfile check.  but I lets not make it more complex.